maps-app-ios/Maps App/App Notifications/LoginNotifications.swift

```diff
@@ -40,12 +40,12 @@ extension MapsAppNotifications {
 
 // MARK: Internals
 extension MapsAppNotifications {
-    static func postLoginNotification(user:AGSPortalUser) {
+    static func postSignInNotification(user:AGSPortalUser) {
         NotificationCenter.default.post(name: MapsAppNotifications.Names.appLogin, object: mapsApp,
                                         userInfo: [LoginNotifications.userKey:user])
     }
 
-    static func postLogoutNotification() {
+    static func postSignOutNotification() {
         NotificationCenter.default.post(name: MapsAppNotifications.Names.appLogout, object: mapsApp)
```

Comment:
Likewise should we also update the name of the notification definition to `MapsAppNotifications.Names.appSignOut`?